src/vcpkg-fuzz/main.cpp

```diff
@@ -16,6 +16,18 @@ using namespace vcpkg;
 
 namespace
 {
+    DECLARE_AND_REGISTER_MESSAGE(FuzzInvalidKind, (msg::value), "example of {value} is 'utf-8'", "invalid kind: {}");
```

Comment:
Are we worried about localizers thinking 'utf-8' has special meaning given the related parts around localization? (I think adding `json`, `platform-expr` like you did below is fine)

Reply:
I'm not, but I'll also point out that localizers shall never look at this file.

Reply:
Fair!